sway-fmt-v2/src/constants.rs

```diff
@@ -14,9 +14,9 @@ pub const DEFAULT_TAB_SPACES: usize = 4;
 pub const DEFAULT_FN_CALL_WIDTH: usize = 60;
 /// Default max width of the args of a function-like attributes before falling back to vertical formatting.
 pub const DEFAULT_ATTR_FN_LIKE_WIDTH: usize = 70;
-/// Default max width in the body of a struct literal before falling back to vertical formatting.
+/// Default max width in the body of a structure literal before falling back to vertical formatting.
 pub const DEFAULT_STRUCT_LIT_WIDTH: usize = 18;
-/// Default max width in the body of a struct variant before falling back to vertical formatting.
+/// Default max width in the body of a structure variant before falling back to vertical formatting.
```

Comment:
Same here except we should call them `fields`, `variants` only apply to `enum` so we can be more broad since it will be used for all structure types

Reply:
Also here this can be `DEFAULT_STRUCTURE_FIELD_WIDTH`